ChangeLog.d/session_export_private.txt

```diff
@@ -0,0 +1,4 @@
+Changes
+    * Mark mbedtls_ssl_session.exported as private. This was an oversight
+      during the run-up to the release of Mbed TLS 3.0. The field was
+      never intended to be marked public.
```

Comment:
I'd rather not create the impression that not-explicitly-private fields are public. In principle, fields are private by default ([“are not considered API compatibility breaks: (…) Removing a field from a structure, unless the field is documented as public.”](https://github.com/ARMmbed/mbedtls/blob/development/BRANCHES.md#backwards-compatibility)).
```suggestion
    * Explicitly mark mbedtls_ssl_session.exported as private. This was an
      oversight during the run-up to the release of Mbed TLS 3.0. The field
      was never intended to be public.
```

Reply:
I assume this means that this PR shouldn't be marked as an API break then - @gilles-peskine-arm @daverodgman

Reply:
Followed your suggestion, @gilles-peskine-arm